content/html/concepts/tables/tables.md

```diff
@@ -20,13 +20,8 @@ Generally, tables are used to display tabular data in particular. In the past, t
 
 ## Example
 
-|      Name      |    Location    |    Category     | Founded |        CEO        |               Info                |
-| :------------: | :------------: | :-------------: | :-----: | :---------------: | :-------------------------------: |
-|   Pied Piper   | Silicon Valley | Cloud Computing |  2014   | Richard Hendricks | A Middle-Out Compression Solution |
-|     Hooli      | Silicon Valley |   Enterprise    |  1997   |   Gavin Bensen    |       Hooli is About People       |
-| Raviga Capital | Silicon Valley | Venture Capital |  2012   |   Peter Gregory   |       Share Only in Success       |
-|     Aviato     | Silicon Valley |     Travel      |  2006   |  Erlich Bachman   |   Software Aggregation Program    |
-|    See Food    | Silicon Valley |     Mobile      |  2016   |     Jian-Yang     |        The Shazam of Food         |
+![image](https://github.com/Moddinu/docs/assets/5288834/1a865d50-a767-4736-8cc4-ff4d68e57615)
```

Comment:
This image is in the wrong place. it needs to be uploaded to the `/media` folder and use the url:`https://raw.githubusercontent.com/Codecademy/docs/main/media/`